### PR TITLE
Complete app features

### PR DESCRIPTION
**Pull request by johnkrator**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,9 +5,322 @@
...
+        </div>
+    </section>
+    <section id="projects">
+        <div>
+            <div class="project-desc">
+                <h1>Projects</h1>
+                <p>Things I’ve built so far</p>
+            </div>
+
+            <div class="project-grid">
```
Comment (Phildeoner) [inline index.html:98]: This section contents should ne centered, and watch the width so you dont have a left to right scroll

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,740 @@
...
+    width: 120px;
+    height: 120px;
+}
+
+/*projects*/
+.project-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 20px;
+}
```
Comment (Phildeoner) [inline css/index.css:133]: For this you can add justify-items center and align-contents center to make the contents centralized

**Review by Phildeoner — APPROVED** [review]
Hi @johnkrator 👋👋, well-done you did a great Job, consider the suggestions below and after that you are good to merge and wrap up this project
Happy Coding😊😊

File: index.html (hunk 1 above)
Comment (johnkrator) [reply index.html:98]: Alright, I will do that this morning

File: css/index.css (hunk 2 above)
Comment (johnkrator) [reply css/index.css:133]: Alright, I will.

**Pull request merged**